Sources/Subscription/Managers/SubscriptionManager.swift

```diff
@@ -64,7 +64,7 @@ public final class DefaultSubscriptionManager: SubscriptionManager {
                 assertionFailure("Trying to setup AppStore where not supported")
             }
         case .stripe:
-            setupForStripe()
```

Comment:
Curious - Why is this no longer required?

Reply:
This was always a no-op. This was initially designed to have symmetry to App Store behaviour but it did not make much sense and we fetch Stripe products ad hoc.